src/toplevel.c

```diff
@@ -766,15 +766,25 @@ JL_DLLEXPORT void jl_method_def(jl_svec_t *argdata, jl_lambda_info_t *f, jl_valu
     f = m->lambda_template; // because jl_new_method makes a copy
     jl_check_static_parameter_conflicts(m, tvars);
 
-    // TODO
```

Comment:
@JeffBezanson added this in afd4eb038dcd9fa3e512509f4c332a9e7763ba59, but I don't see anything that seems to need to-be-done. I'm guessing this is just out-of-date and was already fixed in the same PR?